src/Venturecraft/Revisionable/Revision.php

```diff
@@ -148,7 +148,12 @@ private function getValue($which = 'new')
 
                     // Finally, now that we know the namespace of the related model
                     // we can load it, to find the information we so desire
-                    $item = $related_class::find($this->$which_value);
+                    $item;
+                    if (array_key_exists('Illuminate\Database\Eloquent\SoftDeletes', class_uses($related_class))) {
```

Comment:
Maybe just check for existence of `$related_class::withTrashed()` instead of specific include?

Reply:
Turns out that method checking doesn't really work with included traits in classes:

`>>> method_exists('Jitterbug\Models\AudioVisualItem', 'withTrashed');`
`=> false`
`>>> method_exists('Jitterbug\Models\AudioVisualItem', 'getBatchAttribute');`
`=> true`
`>>> Jitterbug\Models\AudioVisualItem::withTrashed()->find(1234);`
`=> Jitterbug\Models\AudioVisualItem {#918`
     `id: 1234,`
     `call_number: "FT-5882",...`

Reply:
Based on some stackoverflow [scanning](https://stackoverflow.com/questions/36515677/php-instanceof-for-traits), it also seems to be accepted to use the `class_uses` helper array thing to see if a class has a particular trait.